lib/chef/knife/bootstrap_windows_winrm.rb

```diff
@@ -47,8 +47,9 @@ def run
           end
         end
 
-        validate_options
+        validate_options!
         resolve_session_options
+        load_windows_specific_gems if @session_opts[:transport] == :sspinegotiate
```

Comment:
Will be submitting a separate PR moving this dependency loading into `WinrmSession` since it is what actually takes the dependency and that will remove the duplicate calls.